doc/user/sql/create-source/text-source.md

```diff
@@ -45,6 +45,7 @@ _path_ | The absolute path to the file you want to use as the source.
 **WITH (** _option&lowbar;list_ **)** | Options affecting source creation. For more detail, see [`WITH` options](#with-options).
 **REGEX** _regex_ | Format the source's data as a string, applying _regex_, whose capture groups define the columns of the relation. For more detail, see [Regex format details](#regex-format-details).
 **TEXT** | Format the source's data as ASCII-encoded text.
+**AVRO OCF** | Parse the source using as an Avro Object Container File. The file's embedded schema will define the columns of the relation.
```

Comment:
typo around "using as an" I think

Reply:
Actually this seems like maybe the wrong place for this, since this doc is about unstructured text files that you have to hack your way through with a regex. Even CSV files are considered structured enough to be documented elsewhere. Seems like we should do something similar for OCF files?

/cc @sploiselle for his thoughts

Reply:
Yes, that's exactly right.

Reply:
Er, but they're sources, so "Avro from local file". I think that's what you meant! Just wanted to double check.

Reply:
Prepositions are hard :) I was paralleling it with the current links like "Avro over Kafka." Semantics. Nonetheless, @quodlibetor I can take this on.